pyproject.toml

```diff
@@ -0,0 +1,12 @@
+[build-system]
+requires = [
+    "wheel",
+    "setuptools",
+    "Cython>=0.29.18",
+    "numpy==1.15.1; python_version=='3.6' and platform_system!='AIX'",
```

Comment:
Can you add a comment explaining what `AIX` is?

Reply:
Alright, done.

I would think that `platform_system` implies "it's an OS", and people can google it.

Maybe all the AIX lines should go away entirely. Just:

```
    "numpy==1.15.1; python_version=='3.6',
    "numpy==1.15.1; python_version=='3.7',
    "numpy==1.17.3; python_version>='3.8',
]
```

Reply:
I've been googleing to understand what the possible values for `platform_system` are for 3 years now.

I still can't find a comprehensive list....

The IBM pointer is a good one. I'll update it a little bit.

Reply:
I don't know if you want to adopt this or not, but we made a little shim for simplifying this: https://pypi.org/project/oldest-supported-numpy/

Reply:
yeah. might be a good idea. Not too sure about the ramifications of requiring a moving target at build time.

Reply:
I'm not against it, but I feel like it would hurt the chances of this PR getting merged in.

Seems like a good followon PR to me.

Reply:
For reference, the reason for requiring NumPy 1.16 is: https://github.com/scipy/scipy/pull/10431#issue-295816438